src/NAppUpdate.Tests/Conditions/BooleanCondition.cs

```diff
@@ -1,12 +1,13 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-using NAppUpdate.Framework.Conditions;
+﻿using NAppUpdate.Framework.Conditions;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 
 namespace NAppUpdate.Tests.Conditions
 {
+	using Xunit;
```

Comment:
The only real change to the test project was moving it across to XUnit. I have no preference on net core test frameworks but I know XUnit works.